Gemfile

```diff
@@ -1,14 +1,13 @@
 source "https://rubygems.org"
+
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
-git_source(:bc)     { |repo| "https://github.com/basecamp/#{repo}" }
+git_source(:basecamp) { |repo| "https://github.com/basecamp/#{repo}" }
 
-# Specify your gem's dependencies in mission_control-jobs.gemspec.
 gemspec
 
-gem "sqlite3"
-
-gem "sprockets-rails"
-gem "solid_queue", bc: "solid_queue", require: false
-gem "rubocop-37signals", bc: "house-style", require: false
-gem "puma"
 gem "capybara", github: "teamcapybara/capybara"
+gem "puma"
+gem "rubocop-37signals", basecamp: "house-style", require: false
+gem "solid_queue", basecamp: "solid_queue", require: false
+gem "sprockets-rails"
+gem "sqlite3"
```

Comment:
Cosmetic changes unrelated to the PR; please omit

Reply:
No problem